react/BarButton/Readme.md

```diff
@@ -0,0 +1,48 @@
+# BarButton
+This component is used to display a back Button into the Cozy Bar.
+
+### Example
+
+```jsx static
+  const { BarLeft } = cozy.bar
+  return (
+    <BarLeft>
+      <BarButton />
+    </BarLeft>
+    )
+```
+
+## Props
```

Comment:
Note that you can document the prop directly with comments https://react-styleguidist.js.org/docs/documenting.html#code-comments-and-proptypes

Reply:
Yes please.